docs/product/roadmap.md

```diff
@@ -0,0 +1,65 @@
+Product roadmap
+===================
+
+## Overview
+This is a high-level overview. For more detail, see our [story map](https://www.google.com/url?q=https://app.mural.co/t/gsa6/m/gsa6/1647625636085/7825fa5ead63ebb1b65eefab5ea20d24a8ab7c59?wid%3D0-1647626493256%26sender%3Dua4d37dfba3f1e69e09078790&sa=D&source=docs&ust=1654713307298233&usg=AOvVaw2VRKptBGcaZgHol3V6vXpw) or [user flows chart](https://app.mural.co/t/gsa6/m/gsa6/1654109929324/5da173ea5f7ba75104e4e624451e58496a7164c8?sender=ua4d37dfba3f1e69e09078790). As with all agile product roadmaps, this is subject to change. The team revisits and adjusts the roadmap regularly. 
+
+## Product Vision
+* Fully transition the FAC PMO from Census to TTS
+* Transition the [current Census FAC product](https://facweb.census.gov/) into something TTS can maintain
+* Reduce risk and prove value by delivering incremental product updates 
+
+## Release: MVP
+**Goals:** Launch a new FAC intake experience for at least some of current FAC customers on or around 10/1 to begin accepting audits for FY2022. Per the discussed and approved [migration plan](https://github.com/GSA-TTS/FAC/blob/main/docs/product/decisions/0004-P-migration-plan.md), the idea is to keep the old Census FAC running and import FY2022 submissions into that for data disseminination. This will help limit scope and reduce risk of the MVP. 
+
+**Success Metrics (proposed):** 
+* Number of successful FY2022 submissions compared to previous time frame
+* Number of support inquiries compared to previous time frame
+
+**Key Features:**
+* ✅ Technical foundation / infrastructure
+* (80% ✅) Create a new audit (pre SAC forms)
+* (in design) Submit an audit package (post SAC forms)
```

Comment:
Do we need a line for "SAC forms"? We have pre and post, but not the actual SAC tracked here. I'd say SAC forms are 10% done.

Reply:
Added